cmd/network.go

```diff
@@ -204,6 +204,10 @@ func NewAPIRequest(r *Request, api string, args []string, isAsync bool) (map[str
 		}
 	}
 	params.Add("response", "json")
+	params.Add("signatureversion", "3")
```

Comment:
Should we define these keys as constants?

Reply:
Did we want to make a constants file that stores all the constants used? I don't see that in the code currently.

Reply:
Upto you @kevin-lii 
I wouldn't mind adding some constants in the same file itself